File: app/Models/Paciente.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Model;
use App\Models\Usuario;
use App\Models\PersonalSanitario;
use App\Models\Quizs;
use App\Models\Video;

class Paciente extends Usuario
{
    protected $fillable = [
        'estado_enfermedad',
        'estado_animo',
        'actividad_fisica',
        'diabetico',
        'hipertenso',
        'estadio',
        'puntos',
        'personal_sanitario_id',
        'DNI',
        'fecha_nacimiento',
        'peso',
        'altura',
        'IMC',
        'clasificacion',
    ];

    public function personal_sanitario()
    {
        return $this->belongsTo(PersonalSanitario::class);
    }

    public function quizs()
    {
        return $this->belongsToMany(Quiz::class, "quiz_pacientes");
    }

    public function videos()
    {
        return $this->belongsToMany(Video::class, "paciente_videos");
    }
}

File: APP/NefroSalut/app/src/main/java/com/mariona/nefrosalut/models/Paciente.kt
package com.mariona.nefrosalut.models

import com.google.gson.annotations.SerializedName
import java.io.Serializable
import java.time.LocalDate

data class Paciente (
    var id: Long,
    var nombre: String,
    var apellidos: String = "",
    var email: String,
    var telefono: String,
    var password: String,
    @SerializedName("estado_enfermedad")
    var estadoEnfermedad: String,
    @SerializedName("estado_animo")
    var estadoAnimo: String = "",
    @SerializedName("actividad_fisica")
    var actividadFisica: Boolean,
    var diabetico: Boolean,
    var hipertenso: Boolean,
    var estadio: String,
    var puntos: Int,
    @SerializedName("personal_sanitario_id")
    var personalSanitarioId: Int?,
    var DNI: String,
    @SerializedName("fecha_nacimiento")
    var fechaNacimiento: String,
    var peso: Double,
    var altura: Double,
    var IMC: Double,
    var clasificacion: String,
    @SerializedName("progreso_id")
    var progresoId: Int,
): Serializable